docs/getting-started/running/index.md

```diff
@@ -63,10 +63,14 @@ To run the UI tests, use the following command:
 yarn --cwd web-ui test
 ```
 
-Or simply `cd` to the `web-ui` directory and run `yarn test`.
+Or simply `cd` to the `web-ui` directory and run `yarn test`. Tests using Snapshots are likely to fail with component updates. Those which fail to meet their Snapshot will be marked as failed. You can update the Snapshot by running the following command:
+
+```shell
+yarn --cwd web-ui test -u
+```
```

Comment:
I'm thinking you should also mention the option of just pressing the "u" key after failed snapshot tests are reported when the tests are run with "yarn test" (runs in watch mode).